src/olympia/blocklist/models.py

```diff
@@ -409,14 +412,15 @@ def _split_guids_full(cls, guids, v_min, v_max, filter_existing):
     @classmethod
     def _split_guids_fake(cls, guids, v_min, v_max, filter_existing):
         # load all the Addon instances together
-        addon_qs = (
-            Addon.unfiltered
-            .filter(guid__in=guids)
-            .order_by('-average_daily_users')
-            .values_list('guid', 'average_daily_users'))
-        addons = [
-            cls.FakeAddon(guid, addon_users) for guid, addon_users in addon_qs]
-        addon_guid_dict = {addon.guid: addon for addon in addons}
+        addons_qs = Addon.unfiltered.filter(guid__in=guids).values_list(
+            'guid', 'average_daily_users', named=True)
+
+        addons = sorted(
+            addons_qs,
+            key=lambda addon: addon.average_daily_users,
+            reverse=True)
+        adu_lookup = {
```

Comment:
Same question as above. How is this new code different from what was there before?

Reply:
same as above.  The `adu_lookup` is a change though